emacs/lsp-cquery.el

```diff
@@ -293,10 +304,11 @@
       (expand-file-name (locate-dominating-file default-directory "clang_args"))
       (user-error "Could not find cquery project root")))
 
-(lsp-define-stdio-client lsp-cquery "c++"
-                         #'cquery//get-root
-                         (list (concat cquery/root-dir "build/app") "--language-server")
-                         :initialize #'cquery//initialize-client)
+(lsp-define-stdio-client
+ lsp-cquery "c++" #'cquery//get-root
+ (list (concat cquery/root-dir "build/app") "--language-server")
```

Comment:
Consider making this configurable? `(concat cquery/root-dir "build/app") `
If the user does not specify trailing `/` in `cquery/root-dir`, this won't work. What about `(file-name-directory cquery/root-dir)`?

Reply:
Yeah, we should probably go back to `cquery/executable` and `cquery/resourceDirectory` as separate options